docs/source/release/v6.6.0/framework.rst

```diff
@@ -7,28 +7,33 @@ Framework Changes
 
 New Features
 ------------
-- the approach for inverting large tridiagonal matrices has been improved so that the analytical method can still be used even for large matrices, resulting in a performance improvement. This type of matrix inversion is used when calculating the interpolation errors when performing spline interpolation
+- The approach for inverting large tridiagonal matrices has been improved so that the analytical method can still be used even for large matrices, resulting in a performance improvement. This type of matrix inversion is used when calculating the interpolation errors when performing spline interpolation.
```

Comment:
Just a thought, but would sentence:

> This type of matrix inversion is used when calculating the interpolation errors when performing spline interpolation.

Read better as:

> This type of matrix inversion is used when calculating the interpolation errors while performing spline interpolation.

Reply:
Yes, I like that change, thank you for suggesting.

The headings stay there until we are sure no more release notes will be added; so removing them is one of the last things to do before release.